pkg/prompts/prompts.go

```diff
@@ -244,3 +272,40 @@ func Select[T any](label string, items []T, opt *SelectOpt[T]) (T, error) {
 
 	return items[i], nil
 }
+
+func getCurrentDirName() (string, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		log.Fatalf("getting current directory: %v", err)
+	}
+	return filepath.Base(dir), nil
+}
+
+// Sanitize the directory name to comply with k8s label rules
+func sanitizeAppName(name string) string {
+	// Remove all characters except alphanumeric, '-', '_', '.'
+	reg := regexp.MustCompile(`[^a-zA-Z0-9-_.]+`)
```

Comment:
regex will work here, but it may be overkill when we can use more readable methods like [IsDigit](https://pkg.go.dev/unicode#IsDigit), or at least we can use those in the validator